lib/spack/spack/install_test.py

```diff
@@ -110,6 +111,105 @@ def get_test_suite(name):
     return names[0]
 
 
+def package_class(spec_or_pkg):
+    """Return the package class for the spec or package.
+
+    Args:
+        spec_or_pkg (spack.spec.Spec or spack.package_base.PackageBase): spec or
+            package (class)
+
+    Raises: ValueError: If not given a spec, package, or class
+    """
+    if isinstance(spec_or_pkg, spack.spec.Spec):
+        spec = spec_or_pkg
+        try:
+            cls = spec.package.__class__
+        except AssertionError:
+            try:
+                cls = spec.package_class
+            except spack.repo.UnknownPackageError as e:
+                tty.debug("{0}: Cannot retrieve class from spec: {1}".format(spec.name, str(e)))
+                return None
+
+    elif isinstance(spec_or_pkg, spack.package_base.PackageBase):
+        cls = spec_or_pkg.__class__
+
+    elif inspect.isclass(spec_or_pkg):
```

Comment:
Would this:
```suggestion
    elif issubclass(spec_or_pkg, spack.package_base.PackageBase):
```
work too?